tests/suites/test_suite_psa_crypto.data

```diff
@@ -6981,27 +6981,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
 PSA generate key: RSA, 512 bits, good, sign (PKCS#1 v1.5)
-depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME
+depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME:MBEDTLS_RSA_MIN_KEY_SIZE <= 512
```

Comment:
In PSA tests and code, we're moving away from legacy `MBEDTLS_xxx` constants. So this new work should not introduce `MBEDTLS_RSA_MIN_KEY_SIZE` here. Instead, it should use `PSA_VENDOR_RSA_MIN_KEY_BITS`, which we should add to `include/psa/crypto_sizes.h` (like there's a `PSA_VENDOR_RSA_MAX_KEY_BITS` already). The default value of that should be `MBEDTLS_RSA_MIN_KEY_SIZE` if it's set, and 1024 otherwise.

Reply:
added a config specefic to psa. the problem with having 2 config is if `PSA_VENDOR_RSA_MIN_KEY_BITS` is less than 1024 and `MBEDTLS_RSA_MIN_KEY_SIZE` is not set in the config file (meaning it will default to 1024 due to the definition in rsa.c) it will return invalid input even though the key size is more than or equal what the user configured the library to but I don't see a solution to this without using `MBEDTLS_RSA_MIN_KEY_SIZE` in PSA code.

Reply:
Hmmm. I guess we should set `PSA_VENDOR_RSA_MIN_KEY_BITS` from `MBEDTLS_RSA_MIN_KEY_SIZE` when `MBEDTLS_PSA_BUILTIN_KEY_TYPE_RSA_PUBLIC_KEY` is enabled?